### DISCUSSION TITLE
relay failed issue

### DISCUSSION THREAD

**Discussion by 21pages** [rustdesk/rustdesk#6434] [General]
#A
> Same problem only on a single site for the moment
> 
> The remote client is connected and in Ready state
> ![1](https://github.com/rustdesk/rustdesk/assets/8618744/d8899e95-631a-4f91-87f1-69e7d612c613)
> 
> the local client is always connecting 
> 
> https://github.com/rustdesk/rustdesk/assets/8618744/30c78add-e60d-4101-9c13-a26aeedcbf08

#B
> The last time I was able to connect was version 1.2.1

#C
>This is what is present into log of hbbr for this connection
```
[src/relay_server.rs:456] New relay request 14644c4c-1316-4663-bd4e-3bdb7cf1e0ce from [::ffff:xx.x30.1xx.2xx]:30919
[src/relay_server.rs:452] Relay of [::ffff:xx.x30.1xx.2xx]:32253 closed

Normally there are

[src/relay_server.rs:456] New relay request 909c09b0-d8fd-4b8d-a18f-be68f37cd0d2 from
[src/relay_server.rs:440] Relayrequest 909c09b0-d8fd-4b8d-a18f-be68f37cd0d2 from [::ff
[src/relay_server.rs:446] Both are raw
```

#D
> The log is not complete and I cannot post here for security. The version is, as told, 1.2.3
> 
> Yes, aleays get "deadline". In the past (1.2.1 client and 1.1.9 server, i have license), after a couple of attempts it was possible to connect
> 
> ![image](https://github.com/rustdesk/rustdesk/assets/8618744/eb66480b-fd43-4c68-add4-7204be30324e)

both client are configured to use the same relay server. Connecting with ip address, instead, is working.
@pdifeo

**Comment by 21pages** [comment]
@pdifeo Which version of hbbs are you using?

**Reply by pdifeo** [reply]
![image](https://github.com/rustdesk/rustdesk/assets/8618744/be58716e-9cd5-4fb7-b9a4-042bcf3034c9)

```
[root@rustdesk bin]# hbbs -V
hbbs 1.1.10-5
```

```
[root@rustdesk bin]# hbbr -V
hbbr 1.1.10-5
```

**Reply by 21pages** [reply]
Relay will always fail except connecting with ip ?

**Comment by pdifeo** [comment]
> Relay will always fail except connecting with ip ?

Yes. Let me be clear, this happens with all computers at a specific site. Other sites (other customers) work correctly.

**Reply by rustdesk** [reply]
It seems this site can not connect to the relay server.

**Reply by rustdesk** [reply]
Or some network issue like this, https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet

**Comment by pdifeo** [comment]
> Or some network issue like this, https://github.com/rustdesk/rustdesk/wiki/FAQ#deploy-rustdesk-server-in-intranet

The server isn't in-intranet. Is on Internet

Maybe you are right for connection problem. This is the result from nmap


PORT      STATE    SERVICE
21114/tcp open     unknown
21115/tcp open     unknown
21116/tcp open     unknown
21117/tcp filtered unknown
21118/tcp open     unknown
21119/tcp filtered unknown

Ports 21117 and port 21119 results filtered and not opened. On other site, all ports results opened . I'll investigate ... but very strange

**Reply by pdifeo** [reply]
Please close the thread. I solved it. There was an incorrect rule on the customer's firewall.

Thank you very much for your support.

**Reply by pdifeo** [reply]
Just a small note. It would have been useful for me to realise that the relay server was not contactable. The fact that I could see the status "Ready" misguided me.

**Reply by alexander-krug** [reply]
@pdifeo: **"Ready" refers to the connection to the hbbs "ID" server**, which the client uses to get its numbered ID. 
**"relay" server is a different binary (hbbr)**, though often on the same server. (Also see: https://github.com/rustdesk/rustdesk-server/blob/master/README.md#how-to-build-manually)
That's why "Ready" is indeed misleading. 
_(I hope this information helps other future readers of this thread, that's why I added these details)_

**Reply by pdifeo** [reply]
As matter of fact, I wrote " It would have been **useful** for me to realise that the relay server was not contactable."
